rest/model/task.go

```diff
@@ -264,6 +263,12 @@ func (at *APITask) BuildFromService(t interface{}) error {
 			}
 		}
 
+		if v.TimeTaken != 0 {
+			at.TimeTaken = NewAPIDuration(v.TimeTaken)
+		} else if v.Status == evergreen.TaskStarted {
+			at.TimeTaken = NewAPIDuration(time.Duration(time.Since(v.StartTime).Nanoseconds()))
+		}
```

Comment:
Is this nanoseconds conversion needed? time.Since returns a Duration. it seems like you are converting a duration to nanoseconds then back to a duration.